pkg/lua/Makefile.lua

```diff
@@ -0,0 +1,9 @@
+SRC := $(filter-out lua.c luac.c,$(wildcard *.c))
+
+# This builds for native using POSIX system calls and some extra libraries, and
+# removes a compiler warning that warns against using tmpnam().
+ifeq ($(BOARD),native)
+CFLAGS += -DLUA_USE_LINUX
```

Comment:
minor, this should be indented with 2 spaces

Reply:
A frustrated Dan just came to my office. Can we stop asking for minor changes when a PR is already ACK'd, please? This is also something that scares away contributors, IMHO. If the PR is defective and the ACK was given wrongly, yes then asking for changes is definitely okay. But not like this! Yes code quality is important, but these kind of non-functional errors can also be fixed in post.

Reply:
I was pinged for a build issue, so I came, had a look at the package makefile, saw this and finally decided to **comment** this. I'm not requested any change.

Reply:
Yep, and I guess @miri64 hit the point: Reviewers should carefully work with authors to get things into a state at which code can be merged. For this reviews need structuring, streamlining, and pacing in the most constructive manner. 
This means for instance that code-style review should be completed at some early stage, but also means to aid authors in case of troubles with build or similar.
I guess that's a generic discussion, but a very important one: I've seen many old open PRs where authors have been exhausted by being pushed around randomly without hitting anywhere.

Reply:
I mean, I was just going to make the changes, as it would have been quicker than having a discussion about it. But the impression I get is that the PRs building up is a real problem and the review process itself should a) allow for valuable support of contributors by maintainers to get it into a good shape, which tbh has formed the bulk of my time on this PR; but also b) try not to duplicate work and streamline the process otherwise. I think it would be worth talking about tomorrow

Reply:
> I was pinged for a build issue, so I came, had a look at the package makefile, saw this and finally decided to comment this. I'm not requesting any change.

Just to be clear, I appreciate that you only *commented* on this and did not make a change request.

> I didn't know an ACK by one reviewer should stop others from pointing out errors or possible improvements (as minor as they might be) anyway.

I already pointed out, that it's totally fine if the PR is defective, a late comment is ok. But it certainly gives a weird impression if a person that wasn't involved in the discussion before suddenly comes in and also comments on minor, non-functional details, when the PR was already ACK'd (even if they were pinged in for a completely different issue).

Reply:
I think it's worth talking about this over voice call tomorrow... not sure this is a great format for this kind of discussion... :)